native/src/modules/navigation/InternalPathnameParser.js

```diff
@@ -0,0 +1,199 @@
+// @flow
+
+import {
+  CATEGORIES_ROUTE,
+  DASHBOARD_ROUTE,
+  DISCLAIMER_ROUTE,
+  EVENTS_ROUTE,
+  LANDING_ROUTE,
+  LOCAL_NEWS_TYPE,
+  NEWS_ROUTE,
+  OFFERS_ROUTE,
+  POIS_ROUTE,
+  SEARCH_ROUTE,
+  SPRUNGBRETT_OFFER_ROUTE,
+  TU_NEWS_TYPE
+} from 'api-client/src/routes'
+import type {
+  LocalNewsType,
+  TuNewsType
+} from 'api-client/src/routes'
+import type { RouteInformationType } from './createNavigate'
+
+const FEATURE_ID_INDEX = 3
```

Comment:
What does Feature ID mean here? It is used as type in some places. Is it the route type?

Reply:
No, it is the id of a single event/poi/..., e.g. for `/augsburg/de/events/1234` it is `1234.

Reply:
Minor: So `ENTITY_ID_INDEX`?

Reply:
Will do in #135